### ISSUE TITLE
This repo is a scam

### ISSUE THREAD

**Issue by VladyslavZavadskyi** [salihkaragoz/pose-residual-network-pytorch#14]
Don't waste your time dealing with the code, cause I did it for you.
I've made an input-label collage. As you can see the network learns to blur the (slightly blurred and filtered) labels passed to it as input.
![io](https://user-images.githubusercontent.com/16824835/45743677-439cad80-bc05-11e8-8734-56022a8f57c4.png)

**Comment by VladyslavZavadskyi** [comment]
Model itself looks like this
```python
class PRN(nn.Module):
    def __init__(self,node_count,coeff):
        super(PRN, self).__init__()
        self.flatten   = Flatten()
        self.height    = coeff*28
        self.width     = coeff*18
        self.dens1     = nn.Linear(self.height*self.width*17, node_count)
        self.bneck     = nn.Linear(node_count, node_count)
        self.dens2     = nn.Linear(node_count, self.height*self.width*17)
        self.drop      = nn.Dropout()
        self.add       = Add()
        self.softmax   = nn.Softmax(dim=1)

    def forward(self, x):
        res = self.flatten(x)
        out = self.drop(F.relu(self.dens1(res)))
        out = self.drop(F.relu(self.bneck(out)))
        out = F.relu(self.dens2(out))
        out = self.add(out,res)
        out = self.softmax(out)
        out = out.view(out.size()[0],self.height, self.width, 17)

        return out
```

**Comment by mkocabas** [comment]
Hey @VladislavZavadskyy,

Could you clearly describe your problem instead of making a groundless judgment? This repo isn't a full pipeline of the things that we introduced in our recent paper, just a demo of the main contribution to help people to understand the idea. If you can state your issue in a concise way, maybe we can guide you to correct resources.

Thanks,

**Comment by VladyslavZavadskyi** [comment]
Your main contribution is 3-layer perceptron trained on ground truth labels. With results existing only as numbers in the paper and "the rest of the network" being unavailable due to "some license problems" it's clear enough for me. 
Thanks for trying though.

**Comment by sjiang17** [comment]
@VladislavZavadskyy Hi, have you tried what is it like for input heatmaps with multiple peaks?
